sources/Application/Views/ProjectView.cpp

```diff
@@ -119,9 +119,15 @@ ProjectView::ProjectView(GUIWindow &w,ViewData *data):FieldView(w,data) {
     v = project_->FindVariable(VAR_SOFTCLIP);
     position._y += 1;
     UIIntVarField *f3 =
-        new UIIntVarField(position, *v, "soft clip: %s", 0, 4, 1, 1);
+        new UIIntVarField(position, *v, "soft clip: %s", 0, 4, 1, 3);
     T_SimpleList<UIField>::Insert(f3);
 
+    v = project_->FindVariable(VAR_CLIP_ATTENUATION);
+    position._x += 18;
+    f3 = new UIIntVarField(position, *v, "post: %d", 1, 100, 1, 10);
```

Comment:
I think it would be better to have the default of "no attenuation", 100, and make it work regardless if softclip is turned on or not; I'd also put it after the hardclip part, since it should never cause any additional clipping, and I do drive the hardclip for effect, it works in small doses

Since it is now a de-facto master volume, and current master volume is drive/gain/etc. maybe it'd be better to rename current "master" to "drive"/"gain"/etc. and "post" to "master"/"out". 

What do you think?